inboxes/migrations/0002_auto_20220329_2124.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.6 on 2022-03-30 03:24
```

Comment:
this change needs a migration (renaming table col). next time please include migration file 👍